src/components/NavBar/NavbarDesktop.jsx

```diff
@@ -61,9 +62,14 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
-          <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-            PASS
-          </Typography>
+          <Link href="/">
+            <img
+              src="src/assets/pass-logo.png"
```

Comment:
Static image for logo not loading/rendering in production. Likely it's not being sourced correctly.